shell/common/switches.h

```diff
@@ -215,6 +215,12 @@ DEF_SWITCH(EnableSkParagraph,
            "enable-skparagraph",
            "Selects the SkParagraph implementation of the text layout engine.")
 
+DEF_SWITCH(RetainVM,
+           "retain-vm",
```

Comment:
Perhaps a pet peeve of mine, but the "retain" may indicate that that there is a way to release the VM later. There is no such feature. Also, calling this retain-vm in one place and the same thing leak-vm in another is inconsistent and potentially confusing. I would just call it leak-vm like the shell switch.

Reply:
Reasonable & agree with you. Keep them all called "leak" as same ,changed these var to leak_vm/ LeakVM. Done